### PR TITLE
Feat/lesson01

### PR DESCRIPTION
**Pull request by lewtun**

### CODE REVIEW

File: notebooks/lesson01_intro-to-pandas.ipynb [hunk 1]
```diff
@@ -0,0 +1,1487 @@
...
+    "# Lesson 1 - Introduction to pandas and seaborn\n",
+    "\n",
+    "> How to use the pandas and seaborn libraries to quickly load data into Python, calculate descriptive statistics, and make simple visualisations."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/lewtun/dslectures/master?filepath=notebooks%2Flesson01.ipynb)"
```
Comment (lvwerra) [inline notebooks/lesson01_intro-to-pandas.ipynb:16]: The binder badge seems to point to the old lesson_01 notebook not the lesson_01_intro.. notebook.

**Review by lvwerra — COMMENTED** [review]
* What do you think about having a section "Learning goals" at the beginning of the notebooks?
* Also I find the meme density questionable ;)

Otherwise it looks good!

File: notebooks/lesson01_intro-to-pandas.ipynb (hunk 1 above)
Comment (lewtun) [reply notebooks/lesson01_intro-to-pandas.ipynb:16]: Good catch!

**Comment by lewtun** [comment]
> * What do you think about having a section "Learning goals" at the beginning of the notebooks?
> * Also I find the meme density questionable ;)
> 
> Otherwise it looks good!

Good idea about the learning goals. Would you prefer if we remove the memes?

**Pull request merged**